### PR TITLE
Redirect all trailing slashes locally for consistency

### LINKED ISSUE #1991: Do trailing-slash redirects locally for consistency
**Is your feature request related to a problem? Please describe.**
Mozart (or some other upstream service) is doing 301 trailing slash redirects for us - see below.
image
Simorgh doesn't.

**Describe the solution you'd like**
Simorgh does.

A quick search suggests some short express middleware like <https://stackoverflow.com/questions/13442377/redirect-all-trailing-slashes-globally-in-express>. Do some more research and decide if that is a decent solution or if there is a better one.

- *Confirm with other pods before doing this*
 
**Describe alternatives you've considered**
Simorgh continues to not.

**Testing notes**
[Tester to complete]

Dev insight: Unit tests, consider a cypress tests for local OR all envs (testing mozart behaviour)

**Additional context**

### PR DESCRIPTION
**Pull request by RayNjeri**
Signed-off-by: Rachael Njeri <rachael.njeri@andela.com>

Resolves [#1991](https://github.com/bbc/simorgh/issues/1991)

**Overall change:** _Added a middleware that redirects all trailing slashes eg `http://localhost:7080/igbo/ `to `http://localhost:7080/igbo`._

**Code changes:**

- _Added a middleware to the routes._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/server/index.jsx [hunk 1]
```diff
@@ -40,6 +40,13 @@ class LoggerStream {
 
 const server = express();
 
+server.use((req, res, next) => {
+  const urlPath = /\?[^]*\//.test(req.url);
```
Comment (dr3) [inline src/server/index.jsx]: Do we need `urlPath`? https://amitd.co/code/javascript/strip-trailing-slashes-from-express-requests as an example doesnt use it?

Also can you move this to `if (process.env.APP_ENV === 'local') {` please, as it should be local only :)

Comment (dr3) [inline src/server/index.jsx]: Do we need `urlPath`? https://amitd.co/code/javascript/strip-trailing-slashes-from-express-requests as an example doesnt use it?

Also can you move this to `if (process.env.APP_ENV === 'local') {` please, as it should be local only :)

**Review by dr3 — CHANGES_REQUESTED** [review]

File: src/server/index.jsx (hunk 1 above)
Comment (RayNjeri) [reply src/server/index.jsx]: Noted.

**Review by twinlensreflex — APPROVED** [review]
looks good to me

File: src/server/index.jsx [hunk 2]
```diff
@@ -73,6 +73,12 @@ if (process.env.APP_ENV === 'local') {
     });
   };
 
+  server.use((req, res, next) => {
```
Comment (dr3) [inline src/server/index.jsx]: Tiny thing, but could you put this in the chain of them on line 82, they should all chain off each-other

`service.use().get().get()` etc

**Review by dr3 — CHANGES_REQUESTED** [review]

**Review by dr3 — APPROVED** [review]

**Comment by david-boydell** [comment]
Looks good, thanks. This is ready for merge.

**Pull request merged**